docs/logs/using.asciidoc

```diff
@@ -78,6 +78,13 @@ This opens the *Log event document details* fly-out that shows the fields associ
 To quickly filter the logs stream by one of the field values, in the log event details, click the *View event with filter* icon image:logs/images/logs-view-event-with-filter.png[View event icon] beside the field.
 This automatically adds a search filter to the logs stream to filter the entries by this field and value.
 
+[float]
+=== View log anomalies
+If you have a license that includes the machine learning features, you can click *Analysis* to <<xpack-logs-analysis-page, use machine learning to detect and inspect anomalies>> in your log data.
```

Comment:
I would remove the licensing content and replace with something like:

To use machine learning for detecting and inspect anomalies in your log data, click **Analysis**.